lib/db/mysql.js

```diff
@@ -540,6 +540,63 @@ module.exports = function (log, error) {
     )
   }
 
+  // VERIFICATION REMINDERS
+
+  // INSERT : id, uid, email, emailCode, type, acceptLanguage, createdAt
+  var CREATE_REMINDER = 'CALL createVerificationReminder_1(?, ?, ?, ?)'
+
+  MySql.prototype.createVerificationReminder = function (request) {
+    var reminderData = request.body
+    if (! reminderData || ! reminderData.uid || ! reminderData.type) {
+      throw new Error('"uid", "type" are required')
+    }
+
+    reminderData.id = crypto.createHash('sha256').update(reminderData.uid + reminderData.type).digest()
+    reminderData.uid = new Buffer(reminderData.uid)
+    reminderData.createdAt = Date.now()
+
+    return this.write(CREATE_REMINDER, [
+      reminderData.id,
+      reminderData.uid,
+      reminderData.type,
+      reminderData.createdAt
```

Comment:
As far I see the pattern in the servers is to use `Date.now()` for createdAt and not use the db.